main.go

```diff
@@ -118,14 +119,19 @@ func listPRs(client *github.Client) []*github.PullRequest {
 					Sort:      "updated",
 					Direction: "desc",
 					ListOptions: github.ListOptions{
-						Page: page,
+						Page:    page,
+						PerPage: 50,
```

Comment:
Based on https://developer.github.com/v3/guides/traversing-with-pagination/#changing-the-number-of-items-received, I think we can set this to `100`.